pymagicc/definitions/__init__.py

```diff
@@ -340,8 +340,8 @@ def get_openscm_replacement(in_var):
             "CH4OXSTRATH2O_RF": "Radiative Forcing|CH4 Oxidation Stratospheric H2O",  # what is this
             "LANDUSE_RF": "Radiative Forcing|Land-use Change",
             "BCSNOW_RF": "Radiative Forcing|Black Carbon on Snow",
-            "CO2PF_EMIS": "Emissions|CO2|MAGICC Permafrost",
-            "CH4PF_EMIS": "Emissions|CH4|MAGICC Permafrost",
+            "CO2PF_EMIS": "Land to Air Flux|CO2|MAGICC Permafrost",
+            # "CH4PF_EMIS": "Land to Air Flux|CH4|MAGICC Permafrost",
```

Comment:
Why commented out now?

Reply:
cause I haven't tested it, I can also just remove if you want

Reply:
Dunno, either add "TODO: Add when tested or needed" or something, or remove it to avoid confusion ...

Reply:
```suggestion
            # "CH4PF_EMIS": "Land to Air Flux|CH4|MAGICC Permafrost",  # TODO: test and then add when needed
```